data/human/culture conversations.txt

```diff
@@ -1627,3 +1627,270 @@ mission "A Sunracer Hovercraft Racing"
 				clear "second place margin"
 				clear "last place margin"
 				clear "hovercraft racing: talked to fan"
+
+
+phrase "three card monte opening"
+	word
+		"You step down from your ship onto the main concourse of the spaceport, "
+		"Strolling past the smoky glow of the spaceport bar, you drift into the main area, "
```

Comment:
"The main area" is a little awkward and unclear. Maybe something more like this would work better?

```suggestion
		"Strolling past the smoky glow of the spaceport bar, you head along the port's main street "
```